alcs-frontend/src/app/shared/header/header.component.scss

```diff
@@ -22,17 +24,43 @@
     height: 100%;
   }
 }
+
 .flex-grid {
   display: flex;
   justify-content: space-between;
-}
-
-.flex-grid .col {
-  width: 80%;
+ 
+  .col {
+    width: 80%;
+  }
 }
 
 .right {
   display: flex;
   align-items: right;
   justify-content: right;
 }
+
+.favorite-btn {
+  visibility: hidden;
+  color: #d3d3d3;
+  
+  mat-icon {
+    margin-bottom: 5px;
+  }
+}
+
+.favorite-btn.gold {
+  color: #ffd700;
+  visibility: visible !important;
```

Comment:
If you change the class to:
.board-item .favorite-btn.gold you shouldn't need important